### PR TITLE
Fix and re-enable intermittent mention-listbox test

### LINKED ISSUE #1891: Intermittent test: mention-listbox "should scroll the selected option into view when opened"
## 🧹 Tech Debt

This test was disabled because it fails intermittently on the pipeline. ~~It passes consistently locally.~~ It fails locally, too, when all tests are run. It passes when run alone. It's unclear why this test is intermittent, but similar tests (named the same) for select and combobox are not.

### PR DESCRIPTION
**Pull request by mollykreis**
# Pull Request

## 🤨 Rationale

Fixes #1891 

It's not clear why, but the mention-listbox test was awaiting `waitForUpdatesAsync` twice before asserting that the scroll position was correct. Other similar tests await `waitForUpdatesAsync` followed by `waitAnimationFrame`. The call to `waitAnimationFrame` is necessary because the scroll is queued with `requestAnimationFrame`.

## 👩‍💻 Implementation

Update the second call to `waitForUpdatesAsync` to a call to `waitAnimationFrame`.

## 🧪 Testing

Ran the test a number of times locally, though I'm not if this verifies much since I was never able to get the test to fail locally before this fix

## ✅ Checklist

- [ ] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

**Comment by mollykreis** [comment]
@m-akinc, will you buddy this PR for me?

**Review by m-akinc — APPROVED** [review]
Fingers crossed

File: change/@ni-nimble-components-bb8ab8b7-633d-4a78-9d87-e9ec1263eaff.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (m-akinc) [inline change/@ni-nimble-components-bb8ab8b7-633d-4a78-9d87-e9ec1263eaff.json:2]: Should be `"none"`, right?

Comment (mollykreis) [reply change/@ni-nimble-components-bb8ab8b7-633d-4a78-9d87-e9ec1263eaff.json:2]:
```suggestion
  "type": "none",
```

**Review by rajsite — APPROVED** [review]

**Pull request merged**